test/web-platform-tests/to-upstream.js

```diff
@@ -1,8 +1,37 @@
 "use strict";
 const path = require("path");
-const readdirRecursive = require("fs-readdir-recursive");
```

Comment:
It is still used by `scripts/webidl/convert.js`

I had to remove it here so that we can get a proper "skipped" notice in karma. I agree generating the manifest is probably easier than trying to do this dynamically while running the tests.